image/store.go

```diff
@@ -325,18 +326,18 @@ func (is *store) imagesMap(all bool) map[ID]*Image {
 		if !all && len(is.children(id)) > 0 {
 			continue
 		}
-		img, err := is.Get(id)
+		img, err := is.Get(ctx, id)
 		if err != nil {
 			logrus.Errorf("invalid image access: %q, error: %q", id, err)
 			continue
 		}
 		images[id] = img
 	}
-	return images
+	return images, ctx.Err()
```

Comment:
Cases like this should not return `ctx.Err()` but `nil`. Handling context is optional and if function has completed, it should not care about it anymore.